apps/console/src/features/identity-providers/components/forms/authenticators/fido-authenticator-form.tsx

```diff
@@ -0,0 +1,270 @@
+/**
+ * Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { TestableComponentInterface } from "@wso2is/core/models";
+import { Field, Form } from "@wso2is/form";
+import isBoolean from "lodash-es/isBoolean";
+import isEmpty from "lodash-es/isEmpty";
+import React, { FunctionComponent, ReactElement, useEffect, useState } from "react";
+import { useTranslation } from "react-i18next";
+import {
+    AuthenticatorSettingsFormModes,
+    CommonAuthenticatorFormFieldInterface,
+    CommonAuthenticatorFormFieldMetaInterface,
+    CommonAuthenticatorFormInitialValuesInterface,
+    CommonAuthenticatorFormMetaInterface,
+    CommonAuthenticatorFormPropertyInterface,
+    CommonPluggableComponentPropertyInterface
+} from "../../../models";
+
+/**
+ * Interface for FIDO Authenticator Form props.
+ */
+interface FIDOAuthenticatorFormPropsInterface extends TestableComponentInterface {
```

Comment:
Do not use `TestableComponentInterface ` replace with `IdentifiableComponentInterface`

Reply:
Resolved via  https://github.com/wso2/identity-apps/pull/4156/commits/8968e9e9e80de6f997577d12f3f6222db508b1de